### PR TITLE
[ISSUE #3860] rocketmq/es log plugin client authorize field and refactor log plugin

### PR DESCRIPTION
**Pull request by damonxue**
Description
rocketmq/es log plugin client authorize field and refactor log plugin.
[ #3860 ](https://github.com/apache/shenyu/issues/3860)

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Comment by damonxue** [comment]
@moremind pls comment or review after checks passed.

**Comment by moremind** [comment]
pls check ci

**Review by yu199195 — DISMISSED** [review]

**Comment by yu199195** [comment]
pls resolve conficts

**Comment by damonxue** [comment]
@yu199195 pls review. Thanks.

**Review by yu199195 — DISMISSED** [review]

**Comment by yu199195** [comment]
pls check ci

File: shenyu-plugin/shenyu-plugin-logging/shenyu-plugin-logging-common/src/main/java/org/apache/shenyu/plugin/logging/common/client/LogConsumeClient.java [hunk 1]
```diff
@@ -40,9 +40,4 @@
      * @param config logClientConfig
      */
     void initClient(C config);
-
-    /**
-     * close the client.
-     */
-    void close();
```
Comment (yu199195) [inline shenyu-plugin/shenyu-plugin-logging/shenyu-plugin-logging-common/src/main/java/org/apache/shenyu/plugin/logging/common/client/LogConsumeClient.java]: why remove this

Comment (damonxue) [reply shenyu-plugin/shenyu-plugin-logging/shenyu-plugin-logging-common/src/main/java/org/apache/shenyu/plugin/logging/common/client/LogConsumeClient.java]: Extends AutoCloseable interface.

**Review by yu199195 — APPROVED** [review]

**Pull request merged**